### PR TITLE
conan center docs

### PR DESCRIPTION
**Pull request by lasote**
- Split bintray page in 2
- Moved to bottom docs (not needed anymore to warn about the migration)
- Kept link for the current website, and other external references.
- Criteria to be reviewed about the inclusion request guides

### CODE REVIEW

**Comment by lasote** [comment]
Pending mention header only libs with no binary and no settings and new -i

File: move_to_bintray.rst [hunk 1]
```diff
@@ -1,5 +1,7 @@
-Conan.io is now on Bintray
-==========================
+.. _mastering:
```
Comment (memsharded) [inline move_to_bintray.rst]: Wrong ref

File: integrations/appveyor.rst [hunk 2]
```diff
@@ -1,3 +1,5 @@
+.. _appveyor_integration:
```
Comment (memsharded) [inline integrations/appveyor.rst]: This might conflict with master docs

File: bintray/move.rst [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    upload packages to it.
+
+  - `conan-center`_
+
+    This repository has moderated, curated and well-maintained packages, and is the place where you can share
+    your packages with the community. To share your package, you upload it to your own (or your organization’s)
+    repositories and submit a request to include it in `conan-center`_. Check :ref:`Working with conan-center<conan_center_flow>`
+
+
+Using the new Bintray repositories
```
Comment (memsharded) [inline bintray/move.rst]: I have already added this section in "creating packages" -> "uploading packages"

**Comment by memsharded** [comment]
I think we might want to re-think the final table of contents.

File: bintray/conan_center_guide.rst [hunk 4]
```diff
@@ -0,0 +1,88 @@
+.. _conan_center_flow:
+
+Contributing packages to conan-center
+=====================================
+
+As a moderated and curated repository, `conan-center`_ will not be populated automatically. Initially, it will be empty.
+To have your recipe or binary package available on `conan-center`_, you need to submit an inclusion request to Bintray,
+and the Bintray team will review your request.
+
+- If you are the author of an open source library, your package will be approved.
```
Comment (lasote) [inline bintray/conan_center_guide.rst]: Highlight

File: bintray/conan_center_guide.rst [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+
+Inclusion guidelines for third party libraries
+**********************************************
+
+In the inclusion request process, the JFrog staff will perform a general review and will make suggestions for improvements or
+better/cleaner ways to do implement the code.
+
+
+Recipe quality
```
Comment (lasote) [inline bintray/conan_center_guide.rst]: Document: If package became undermaintained conan can unlink it.

File: bintray/conan_center_guide.rst [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+--------------
+
+- **Github public repository**: The recipe needs to be hosted in a public Github repository.
```
Comment (lasote) [inline bintray/conan_center_guide.rst]: Git public repository

File: bintray/conan_center_guide.rst [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+- If you are packaging a third-party library you need to follow the guidelines below.
+
+
+Inclusion guidelines for third party libraries
+**********************************************
+
+In the inclusion request process, the JFrog staff will perform a general review and will make suggestions for improvements or
+better/cleaner ways to do implement the code.
+
```
Comment (lasote) [inline bintray/conan_center_guide.rst]: Can't be other conan package for the same library, even for different versions. Contact the author and collaborate.

File: bintray/conan_center_guide.rst [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+CI Integration
+--------------
+
+- Unless your library doesn't support a concrete operating system or compiler you will need to provide a CI systems integration
+  to support:
+
+    - **Linux:** GCC, desirable latest version from each major (4.9, 5.4, 6.3)
+    - **Linux:** Clang
```
Comment (lasote) [inline bintray/conan_center_guide.rst]: **OSX**

File: bintray/conan_center_guide.rst [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+- **Linter:** Is important to have a reasonable clean Linter, ``conan export`` and  ``conan test_package`` will
+  output some warnings and errors, keep it as clean as possible to guarantee a recipe less error prone and more understandable.
+
+- **Updated:** Not using deprecated features and when possible, using latest conan features, build helpers etc.
+
+- **Clean:** The code style will be reviewed to guarantee the readability of the recipe.
+
+- **test_package:** The recipes must contain a :ref:`test_package<packaging_getting_started>`
+
```
Comment (lasote) [inline bintray/conan_center_guide.rst]: - Maintenance commitment

File: bintray/conan_center_guide.rst [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+    - ``.travis`` folder. Not needed to adjust anything.
+    - ``.travis.yml`` file. Adjust your username, library reference etc
+    - ``appveyor.yml`` file. Adjust your username, library reference etc
+
+- Take a look to the :ref:`Travis CI<travis_integration>` and :ref:`Appveyor<appveyor_ci>` integration guides.
+
+
+
+Bintray library page information
```
Comment (lasote) [inline bintray/conan_center_guide.rst]: Bintray package information

File: bintray/conan_center_guide.rst [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+- **Linter:** Is important to have a reasonable clean Linter, ``conan export`` and  ``conan test_package`` will
+  output some warnings and errors, keep it as clean as possible to guarantee a recipe less error prone and more understandable.
+
+- **Updated:** Not using deprecated features and when possible, using latest conan features, build helpers etc.
+
+- **Clean:** The code style will be reviewed to guarantee the readability of the recipe.
+
+- **test_package:** The recipes must contain a :ref:`test_package<packaging_getting_started>`
+
+
```
Comment (lasote) [inline bintray/conan_center_guide.rst]: - **License**: The package have to contain a LICENSE file with the library license.

**Pull request merged**